client/src/pages/Login/Login.tsx

```diff
@@ -14,14 +17,25 @@ import {
   Title,
 } from "./Login.styles";
 
-const Login = () => {
-  const [email, setEmail] = useState<string>("");
-  const [password, setPassword] = useState<string>("");
+const Loginschema = yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup.string().min(8).max(20).required().matches(/^[A-Za-z]+$/i)
+})
 
-  const handleSubmit = (e: React.FormEvent) => {
-    // code to validate login and password
-    e.preventDefault();
-  };
+interface LoginInputs {
+  email: string;
+  password: string;
+}
+
+export default function Login() {
+
+  const { register, handleSubmit, formState: {errors} } = useForm<LoginInputs>({
+    resolver: yupResolver(Loginschema),
+  });
+  // @ts-ignore 
+  const submitForm = (data) => {};
```

Comment:
What is the purpose of this function?

Reply:
I'm guessing this was just left blank for now. This is where all the logic goes

const submitForm:SubmitHandler<LoginInputs> = (data) => {
 >this is where you would add the login API call
}

The SubmitHandler must be imported in the file as well from react-hook-form

Reply:
thats where the fetch request will go sending the data to the api